Dockerfile

```diff
@@ -0,0 +1,12 @@
+# Build
+FROM gradle:4.8.0-jdk10 as build
+COPY build.gradle.kts settings.gradle.kts ./
+COPY src ./src
+RUN gradle shadowJar --no-daemon
+
+# Run
+FROM openjdk:10-jre-slim as runtime
+WORKDIR root
+COPY --from=build /home/gradle/build/libs/app.jar .
+VOLUME /tmp
```

Comment:
I think this optimization is only relevant for SB applications.
- https://dzone.com/articles/running-spring-boot-in-a-docker-container
- https://stackoverflow.com/questions/30464204/spring-boot-docker